source/AsteroidBelt.h

```diff
@@ -0,0 +1,58 @@
+/* AsteroidBelt.h
+Copyright (c) 2014 by Michael Zahniser
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+class DataNode;
+
+
+
+// Class defining the geometry of one asteroid belt, including radius and eccentricity parameters.
+class AsteroidBelt {
+public:
+	AsteroidBelt(double radius) : radius(radius) {}
+	AsteroidBelt(double radius, const DataNode &node);
+
+	double Radius() const;
+	double MaxEccentricity() const;
+	double ScaleFactorMinLow() const;
+	double ScaleFactorMinHigh() const;
+	double ScaleFactorMaxLow() const;
+	double ScaleFactorMaxHigh() const;
+
+
+private:
+	double radius;
+	double maxEccentricity = .6;
+	double scaleFactorMinLow = .4;
+	double scaleFactorMinHigh = .8;
+	double scaleFactorMaxLow = 1.3;
+	double scaleFactorMaxHigh = 4;
```

Comment:
Perhaps it would be better to name these consistently with their data keys. Descriptions like "MaxLow" aren't very clear.